actions/views/profile_popover.js

```diff
@@ -8,21 +8,21 @@ import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
 
 import {getSelectedPost} from 'selectors/rhs';
 
-export function getMembershipForCurrentEntities(userId) {
+export function getMembershipForCurrentEntities(userId, channelId) {
```

Comment:
It looks like `channelId` is optional here, but it seems to only be called in a single place. Can we just make it required instead and skip the `defaultChannelId` logic altogether?

Reply:
I was focusing on least disruptive here, and didn't realize it was only called in one place. We have `currentTeamId` there already, so we don't even need a lot of this—thanks for bringing it up!